### PR TITLE
Frame UI command

### PR DESCRIPTION
**Pull request by jasperdewinther**

### CODE REVIEW

File: code/headers/frame_types.hpp [hunk 1]
```diff
@@ -194,7 +194,27 @@ namespace r2d2 {
...
+     *	module is the name of the targeted module, mostly used to prevent nameclash
+     *	command is the command that needs to be executed
+     *	destination is used to tell what robot to send the command to
+     *	all parameters needed for the command serialised
+    */
+    struct frame_ui_command_s{
+        char module;
+        char command;
+        char destination;
+        char parameters;
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp]: As discussed, `command` and `parameters` can be combined by serializing (Pickle is the standard here: https://docs.python.org/3.7/library/pickle.html) the wrapped frame.

File: code/headers/frame_types.hpp [hunk 2]
```diff
@@ -194,7 +194,27 @@ namespace r2d2 {
...
+     *This conversion does not work in c++
+     *These frames will be sent to swarm management, they could then fill the required frame with parameters and send it to the destined robot
+     *
+     *Params:
+     *	module is the name of the targeted module, mostly used to prevent nameclash
+     *	command is the command that needs to be executed
+     *	destination is used to tell what robot to send the command to
+     *	all parameters needed for the command serialised
+    */
+    struct frame_ui_command_s{
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp]: Add a space after `_s`

**Review by LRstudentHU — CHANGES_REQUESTED** [review]

File: code/headers/frame_types.hpp (hunk 1 above)
Comment (itzandroidtab) [reply code/headers/frame_types.hpp]: Maybe a link to the module as well?

**Comment by LRstudentHU** [comment]
You also need to add the frame helper and enum value, see the readme for details

**Review by LRstudentHU — APPROVED** [review]

**Pull request merged**